rs/tests/consensus/BUILD.bazel

```diff
@@ -366,3 +366,14 @@ system_test_nns(
         "@crate_index//:slog",
     ],
 )
+
+system_test(
```

Comment:
I see `rs/tests/consensus/large_file_resuming_download.rs` is actually not a system-test, i.e. it doesn't setup a testnet. I would suggest moving this test under `[rs/http_utils](https://github.com/dfinity/ic/pull/4415/files#diff-59795d7e7dd973677c0399d80ba6adfe54270a13f1958b3dc7121e6a1deddf70)` and have it has a `rust_test`.

Reply:
I am just concerned that any breaking of this code will be visible only on master. Since we can write a mock to fit the test and not test against an actual server its possible to have some degradation here over time.

Reply:
Note that if you label your PR with `CI_ALL_BAZEL_TARGETS` it will run all tests including the `long_tests`. Let's see how much this will break master. If it's too much we can reconsider running this on PRs.